sdk/nodejs/serverless/function.ts

```diff
@@ -254,55 +270,51 @@ interface Package {
 
 // allFolders computes the set of package folders that are transitively required by the given list of package
 // dependencies rooted in a package at the provided path.
-function allFoldersForPackages(path: string, packages: string[]): Promise<Set<string>> {
+function allFoldersForPackages(includedPackages: Set<string>, excludedPackages: Set<string>): Promise<Set<string>> {
     return new Promise((resolve, reject) => {
-        readPackageTree(path, undefined, (err: any, root: Package) => {
+        readPackageTree(".", undefined, (err: any, root: Package) => {
             if (err) {
                 return reject(err);
             }
+
+            const allPackages = new Set<string>(includedPackages);
```

Comment:
Perhaps just `packages` - since it is not actually "all" of the packages - it's goal is explicitly to pick out a subset of all of the packages that are available under `node_modules`.